src/sdkBase.ts

```diff
@@ -27,6 +27,24 @@ export type SDKSetup = {
 };
 export type CallbackContainer = { callback?: () => void };
 
+export class IdHelper {
```

Comment:
Perhaps `UidHelper` would be a better name? It is the generic term for UID2/EUID: https://unifiedid.com/docs/ref-info/glossary-uid#gl-uid

Reply:
Good callout, I've made this change